docs/suspensive.org/src/pages/docs/react-await/awaitOptions.en.mdx

```diff
@@ -0,0 +1,28 @@
+import { Callout } from '@/components'
+
+# awaitOptions
+
+<Callout type='experimental'>
+
+`<Await/>` is experimental feature, this interfaces could be changed
+
+</Callout>
+
+`awaitOptions` combines a key and a function to help manage the options in the current `<Await />` component.
```

Comment:
If the interface changes in the future, we will also change the document. But I thought it was right to specify the <Await /> component to deliver explicitly as of now